dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -214,11 +238,29 @@
         <Setter Property="HighContrastAdjustment" Value="None" />
     </Style>
 
+    <Style x:Key="InkToolbarPenConfigurationControlTextStyle" TargetType="TextBlock" BasedOn="{StaticResource BodyTextBlockStyle}">
+        <Setter Property="FontWeight" Value="SemiBold" />
+    </Style>
+    
     <Style x:Key="InkToolbarSelectionAccentStyle" TargetType="Rectangle">
```

Comment:
If this is no longer being used, the style should be removed from this file, and all SelectionAccent rectangles should be removed from the markup since they were never be visible.

Reply:
Here SelectionAccent is a blue pill that was visible earlier, it was added and removed couple of times now, so put the related code, now decision of removing it seems to be final so will get rid of this!